DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/applepay/apple-pay-chargeback.md

## Chargebacks

> Since Apple Pay payments are credit card transactions, merchants are [3D Secure](/faq/general/glossary/#3d-secure) protected for customer fraud. In such situations whereby a customer initiates a chargeback, email <retrieval@multisafepay.com> and MultiSafepay will challenge the chargeback on your behalf.

Reviewer: ```suggestion
Since Apple Pay payments are credit card transactions, merchants are [3D Secure](/faq/general/glossary/#3d-secure) protected against fraud. Email requests for MultiSafepay to challenge chargebacks on your behalf to <retrieval@multisafepay.com> 
```
